master/SendImages.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import argparse
+import random
+
+from DataBase.Redis import Redis
+
+class SendImages:
+    def __init__(self) -> None:
+        self.args = self.get_arguments()
+
+    def post(self, url: str, data: str) -> requests.Response:
+        return requests.post(url, json=data)
+
+    def find_worker(self) -> str:
+        redis = Redis()
```

Comment:
You should instantiate your database client only once, and in the constructor of your class.
Also now that you have repository pattern, I recommend to accept it as a parameter in your constructor (Dependency Injection).

```python
def __init__(self, db: Repository) -> None:
    self.db = db
```